src/lib/fhe/__tests__/key-rotation.test.ts

```
@@ -0,0 +1,96 @@
+      if (keyId) {
+        await service.reportKeyCompromise(keyId, 'Test compromise')
+        // Should trigger emergency rotation
+        const newKeyId = service.getActiveKeyId()
+        expect(newKeyId).not.toBe(keyId)
+      }
+    })
+  })
+
+  describe('Error Handling', () => {
+    it('should handle missing environment variables gracefully', () => {
+      delete process.env.HIPAA_MASTER_SECRET
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)